etp-backend/src/main/clj/solita/etp/service/energiatodistus_pdf.clj

```diff
@@ -539,7 +539,7 @@
                {:path [:lahtotiedot :ilmanvaihto :tuloilma-lampotila] :dp 1}
                {:path [:lahtotiedot :lammitys :tilat-ja-iv :lampopumppu-tuotto-osuus] :dp 0 :percent? true}
                {:path [:lahtotiedot :lammitys :lammin-kayttovesi :lampopumppu-tuotto-osuus] :dp 0 :percent? true}
-               {:path [:lahtotiedot :lammitys :tilat-ja-iv :lampohavio-lammittamaton-tila] :dp 1}
+               {:path [:tulokset :lampohavio-lammittamaton-tila] :dp 1}
                {:path [:lahtotiedot :lammitys :lammin-kayttovesi :lampohavio-lammittamaton-tila] :dp 1}]
```

Comment:
Tämä A10 -soluun menevä arvo ei ole missään käytössä, en ihan tiedä kannattaisiko se jättää näin tai esimerkiksi poistaa kokonaan.

Reply:
Vaihdettiin käyttöön sama arvo kuin ylemmässä solussa. xlsx:aan merkittiin kuitenkin UNUSED-nimellä